docs/package.json

```diff
@@ -3,7 +3,8 @@
   "name": "@primer/css-docs",
   "scripts": {
     "lint": "eslint pages src",
-    "test": "./test.sh",
+    "pretest": "npm run sync",
+    "test": "node test/urls.js",
```

Comment:
you have a `exceptions.js` test also that isn't in the npm script. Did you miss it or is it for something else